### PR TITLE
6 New Party Parrots

### PR DESCRIPTION
**Pull request by kbvincent**
blondesassyparrot
bluecluesparrot
darkbeerparrot
magaritaparrot
oriolesparrot

and the infamous:
gentlemenparrot

### CODE REVIEW

File: parrots.json [hunk 1]
```diff
@@ -182,5 +182,29 @@
...
+      "gif": "gentlemenparrot",
+      "name": "A true Gentlemen Parrot"
+    },
+    {
+      "gif": "magaritaparrot",
+      "name": "Fiesta Parrot is getting his drink on, magarita style!"
+    },
+    {
+      "gif": "oriolesparrot",
+      "name": "This parrot loves the orioles so much he turned into one!"
```
Comment (jmhobbs) [inline parrots.json]: Names should be relatively short, so "Orioles Parrot" or "Gentleman Parrot".  Can you rework these to use "tip" for the longer descriptions, which puts a little tooltip popup on the website?

File: parrots.json [hunk 2]
```diff
@@ -182,5 +182,29 @@
...
+      "gif": "darkbeerparrot",
+      "name": "Parrot with a dark beer"
+    },
+    {
+      "gif": "blondesassyparrot",
+      "name": "Sassy Parrot with a Blonde wig"
+    },
+    {
+      "gif": "bluecluesparrot",
+      "name": "Steve from Blue's Clues Parrot"
```
Comment (jmhobbs) [inline parrots.json]: Is there some meme with Steve I don't know about, or do you just enjoy Blues Clues? 😄

Comment (kbvincent) [inline parrots.json]: lol, we have a guy on our team that we constant joke looks like steve from blues clues, so he got his own parrot.

File: parrots.json (hunk 1 above)
Comment (kbvincent) [inline parrots.json]: :+1:

**Comment by lucymhdavies** [comment]
`s/gentlemenparrot/gentlemanparrot/` ?

**Pull request merged**